src/index.ts

```diff
@@ -1697,6 +1733,37 @@ class Spanner extends GrpcService {
     }
     return codec.Struct.fromJSON(value);
   }
+
+  static _verifyDirectedReadOptions(
+    directedReadOptions:
+      | google.spanner.v1.IDirectedReadOptions
+      | null
+      | undefined
+  ) {
+    if (directedReadOptions) {
+      if (
```

Comment:
This will throw an error if customer doesn't have include replicas or exclude replicas. And customer can only pass one so this will always throw an error